app/models/pillar.rb

```diff
@@ -13,6 +13,10 @@ def value(pillar:)
     end
 
     def all_pillars
+      simple_pillars.merge(cloud_worker_pillars)
+    end
+
+    def simple_pillars
```

Comment:
MINOR: maybe `simple_pillars` could be called `local_pillars` or `non_pubcloud_pillars` or sth more descriptive?

Reply:
I chose "simple_pillars" because these pillars follow the simplest path from velum -> salt; no munging required like for the cloud or registry pillars.

I expect this name will be short-lived anyway... someone needs to refactor in order to shut up codeclimate ;-)